README.rst

```diff
@@ -52,3 +52,23 @@ Comparing ``rdopkg patch`` with this ``tar-changes`` script:
 
 .. _rdopkg: https://github.com/softwarefactory-project/rdopkg
 .. _Limited: https://github.com/softwarefactory-project/rdopkg/issues/169
+
+##Understanding branches
+========================
+
+We build RH Ceph Storage in `ceph-*` Git branches.
```

Comment:
Again I forgot this is reStructuredText, so we need to use double ` (backtick) characters. Single backticks are for Markdown.
```suggestion
We build RH Ceph Storage in ``ceph-*`` Git branches.

```